styled/auth.js

```diff
@@ -1,33 +1,87 @@
 import styled from 'styled-components/native';
+import Colors from '../assets/Colors';
 
-export const SignUpContainer = styled.View`
+// TODO @tommypoa: Consider creating re-usable containers taking in props for the 3 Auth screens
+
+// StartUpScreen
+
+export const StartUpContainer = styled.View`
   display: flex;
   justify-content: center;
   align-items: center;
   background-color: #fff;
+  margin-top: 241px;
 `;
 
-export const Input = styled.TextInput`
-  width: 350px;
-  height: 55px;
-  background-color: #008550;
-  margin: 10px;
-  padding: 8px;
-  border-radius: 14px;
-  color: white;
-  font-size: 18px;
-  font-weight: 500;
-  align-content: center;
+export const StartUpTitleContainer = styled.View`
+  margin-left: 58px;
+  margin-right: 47px;
```

Comment:
Try something like `margin: 0 15% 0 15%;` so it won't look off-center on different screen sizes. In general cases like these where you're centering things, avoid using pixel margins.